api-spec/payment-requests-api.yaml

```diff
@@ -525,6 +525,12 @@ components:
               type: string
               format: uri
               example: "https://www.comune.di.prova.it/pagopa/error.html"
+        idCart:
+          type: string
+          example: "id_cart"
+        allCCP:
+          type: boolean
+          example: false
```

Comment:
add description for those fields as done for the ones above

Reply:
did it!